config/database.yml

```diff
@@ -13,5 +13,5 @@ development:
   database: c2_development
 test:
   <<: *default
-  database: c2_test
+  url: postgresql://localhost/c2_test
```

Comment:
Iiiinteresting. One might argue that is the expected behavior, but having blown away a production database that same way, very much a :+1: on being extra careful in preventing people from doing it.